.dockerignore

```diff
@@ -0,0 +1,8 @@
+.github
+.vscode
+.dockerignore
+.env
+.env.test
+Dockerfile
+my-ansible-inventory.yml
+my-ansible-playbook.yml
```

Comment:
All sensible shouts - you might find some other useful ones include:
* `.venv` - it's easy to accidentally copy dependencies in that aren't right for the container
* cache files - `**/__pycache__` is the one I always see hanging around on my machine!